### PR TITLE
#157933426 Build tests for API endpoints.

### PR DESCRIPTION
**Pull request by blac-siren**
**What does this PR do?**
Developing the test cases for the request resource.

**Description of Task to be completed?**
Create a challenge 2 branch from develop branch.
Write test for the **/users/requests** endpoint [Method = GET]
Write test for the **/users/requests** endpoint [Method = POST]
Write test for the **/users/requests/<request_id>** endpoint [Method = GET]
Write test for the **/users/requests/<request_id>** endpoint [Method = PUT]
Write test for the **/users/requests/<request_id>** endpoint [Method = DELETE]

**How should this be manually tested?**
Clone the branch into your local machine
Run the following commands on your terminal

```
https://github.com/blac-siren/Maintenance-Tracker.git
cd Maintenance-Tracker

virtualenv -p python3  venv
source venv/bin/activate

pip install -r requirements.txt
```

Any background context you want to provide?
Uses unittest and nose to run the tests

What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/157933426/comments/190095895

### CODE REVIEW

File: api/app/models/user.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        self.username = username
+        self.email = email
+        self.password_hash = Bcrypt().generate_password_hash(password).decode(
+            'UTF-8')
+
+    def create_user(self):
+        """Save user info into dictionary."""
+        user_details = {
+            self.email: {
+                'username': self.username,
```
Comment (zmwas) [inline api/app/models/user.py]: self.email should not be a dictionary

File: api/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Maintenance Tracker
+
+Maintenance Tracker App is an application that provides users with the ability to reach out to operations or repairs department regarding repair or maintenance requests and monitor the status of their request.
```
Comment (zmwas) [inline api/README.md]: Your readme does not tell me how I will install your app and run the tests.

Comment (blac-siren) [reply api/README.md]: I'm going to work on the feedback. Thank you

File: api/app/models/user.py (hunk 1 above)
Comment (blac-siren) [reply api/app/models/user.py]: I'll change that.

**Pull request merged**